contracts/Market.sol

```diff
@@ -61,9 +61,10 @@ contract Market is IMarket, Ownable, ERC721 {
 	mapping(address => bool) private _signers;
 
 	// Race result constants
-    uint8 internal constant WINNER = 0x01;
-    uint8 internal constant LOSER = 0x02;
-    uint8 internal constant SCRATCHED = 0x03;
+	uint8 public constant NULL = 0x00;
+    uint8 public constant WINNER = 0x01;
```

Comment:
Indent seems off

Reply:
Looks fine to me.  I linted too.
![Uploading image.png…]()